std_msgs/Header header
---
uint8 SUCCEEDED=0
uint8 FAILED=1
uint8 result_status

sensor_msgs/PointCloud2 points
